v2.1/inverted-indexes.md

```diff
@@ -156,6 +158,14 @@ Now, run a query that filters on the `JSONB` column:
 +--------------------------------------+----------------------------------+--------------------------------------------------------------------------+
 (2 rows)
 ~~~
+### Comparisons
+Currently, inverted indexes only support equality. If you require comparisons like `>`, `<=`, etc., you can create an index on a computed column using your json payload, and then create a regular index on that. So if you wanted to write a query where the value of "foo" is greater than three, you would:
```

Comment:
1. Suggest expanding the first part of the first sentence slightly to add context - e.g., something like:

> Currently, inverted indexes only support equality comparisons using the `=` operator.  If you require comparisons using `>`, ...

2. 'json' should be capitalized to 'JSON'.

3. This `### Comparisons` section doesn't really fit here (as a subsection of *Example*).  Suggest moving it up the page so it comes directly after the *Performance* section (keeping the existing `###` level)

Reply:
👍 Done.